docs/src/browser-contexts.md

```diff
@@ -1,12 +1,8 @@
 ---
 id: browser-contexts
-title: "Browser Contexts"
+title: "Isolation"
 ---
 
-<!-- TOC -->
-
-## Browser context
-
 A [BrowserContext] is an isolated incognito-alike session within a browser instance. Browser contexts are fast and
```

Comment:
Plan of rework:
- tests are isolated based on [BrowserContext].
- explain contexts.
- you can create contexts manually if you need multiple of them, adapt an example from https://playwright.dev/docs/test-auth#testing-multiple-roles-together.